pkg/master/master.go

```diff
@@ -226,6 +233,14 @@ func setDefaults(c *Config) {
 		}
 		c.PortalNet = portalNet
 	}
+	if c.NodePorts.Size == 0 {
+		// TODO: Currently no way to specify no NodePort range (do we need to allow this?)
```

Comment:
Just that the user has no way to specify an empty (zero-sized) port range: if they specify one, we overwrite it